dataflow-bigquery/modules/ROOT/pages/job-specification.adoc

```diff
@@ -42,6 +44,15 @@ The `sources` section contains the definitions of the data sources, as a list. A
   1. the source table can have more columns than what you select in the query;
   2. multiple targets can use the same source, potentially filtering it for a different subset of columns.
 
+- `query_temp_project` (string, optional) Specifies the project to store temporary query results, processed by Apache Beam. If not set, Apache
```

Comment:
I'm also not sure if we need to mention Apache Beam, since that's mostly an implementation from the user's perspective. What do y'all think?